app/Http/Controllers/BrandController.php

```diff
@@ -1,39 +1,67 @@
 <?php
 // app/Http/Controllers/BrandController_HR.php
+
 namespace App\Http\Controllers;
 
 use App\Models\Brand;
-use App\Http\Requests\StoreBrandHRRequest;
-use App\Http\Requests\UpdateBrandHRRequest;
+use App\Http\Requests\StoreBrandRequest;
+use App\Http\Requests\UpdateBrandRequest;
+use App\Http\Helpers\ResponseHelper;
+use App\Http\Resources\BrandResource;
 use Illuminate\Http\Request;
+use Exception;
 
 class BrandController extends Controller
 {
     public function index()
     {
-        return response()->json(Brand::all());
+        try {
+            $brands = Brand::all();
+            return ResponseHelper::success(BrandResource::collection($brands), 'Brands retrieved successfully.');
```

Comment:
use successResponse(); instead this need to maintaint same success and error response overall